packages/create-jsx-email/starter/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "jsx-email-starter",
+  "version": "0.0.0",
+  "private": true,
+  "description": "A simple starter for jsx-email",
+  "scripts": {
+    "build": "email create",
+    "dev": "email preview ./templates",
+    "export": "email build ./templates"
+  },
+  "dependencies": {
+    "@jsx-email/all": "2.0.0",
+    "@jsx-email/cli": "1.0.9",
+    "superstruct": "1.0.3"
```

Comment:
we'll want to use `"^` ranges on these dependencies, same with `react` below. otherwise we'll be on the hook for updating this for every patch release.